Specification/4._CAN_bus_transport_layer.md

```diff
@@ -8,7 +8,7 @@ This chapter defines the CAN bus based transport layer of UAVCAN.
 
 ## The concept of transfer
 
-*Transfer* is a momentary act of data transmission between nodes.
+A *Transfer* is a momentary act of data transmission between nodes.
```

Comment:
This sentence is a bit annoying (possibly it is the use of the word "momentary"). I don't understand what it is that this definition helps you achieve. In many cases the word could be replaced with "message" and the sentence would make sense. Perhaps this is an attempt to disambiguate UAVCAN and CAN messages?

Reply:
The word "momentary" can be removed safely. Messages and services are higher level entities that applications can directly work with. A message can be sent. A service can be invoked or responded to.

When a message is sent, it is done by means of emitting a *transfer*. Same applies to the case when a service is invoked or responded to. Hence transfer is not the same as message.

In the context of the terminology used here, CAN has no messages; CAN has frames. A transfer may consist of one or multiple CAN frames.

Reply:
OK, I removed "momentary". Still not completely comfortable with this, but that is probably my limitation, not your reader's.

Reply:
I think you **may* be using "message" to refer to a UAVCAN-level message. You are using Transfer to refer to some higher level abstraction's idea of a message. 
So if an app say wanted to broadcast something then this would all be done in one transfer, even if at the UAVCAN level multiple messages were used ... and at the CAN level multiple frames were used. Is that even close?